x/leveragelp/keeper/keeper.go

```diff
@@ -201,3 +201,14 @@ func (k Keeper) GetWhitelistedAddress(ctx sdk.Context, pagination *query.PageReq
 
 	return list, pageRes, err
 }
+
+// SetHooks set the epoch hooks
```

Comment:
need to change the comment here, these hooks are triggered by leveragelp actions